wp-includes/js/service-worker-navigation-routing.js

```diff
@@ -170,7 +165,7 @@ ERROR_OFFLINE_BODY_FRAGMENT_URL, STREAM_HEADER_FRAGMENT_QUERY_VAR, NAVIGATION_BL
 
 			return stream.response;
 		} else {
-			return navigationCacheStrategy.makeRequest( { request: event.request } )
+			return navigationCacheStrategy.handle( { event, request: event.request } )
```

Comment:
This is ultimately the crux of the fix: passing `event`. When this is done, it allows for `event.preloadResponse` to be accessed:

https://github.com/GoogleChrome/workbox/blob/e33e8dcb2e207d97de07a4df9a013be341d58f7b/packages/workbox-core/src/_private/fetchWrapper.ts#L51-L63